homeworks/vlad.zelenko_undercover-agent/5-rxjs/css/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  font: 400 18px/20px sans-serif;
+  padding: 0;
+  margin: 0;
+  text-align: justify;
+  text-indent: 30px;
+}
+
+.wrapper {
+  max-width: 992px;
+  margin: 120px auto;
+}
+
+.header {
+  height: 100px;
+  padding: 0 10px;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  background: #FF4500;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  transition: top .5s;
+  box-sizing: border-box;
+}
+
+.mobile-menu-btn {
+  height: 20px;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-between;
+  cursor: pointer;
+}
+
+.mobile-menu-btn span {
+  display: block;
+  width: 30px;
+  height: 3px;
+  background: #000;
+}
+
+.active-menu {
```

Comment:
You have the **header** class on the **header** tag. After scrolling, your **header** tag with **header** class receives the **active-menu** class. 
So please, rename a class to match "menu" or "header" 😄